src/gui/base/OfflineIndicator.ts

```diff
@@ -11,10 +10,10 @@ export const enum OfflineIndicatorState {
 }
 
 export type OfflineIndicatorAttrs =
-	| { state: OfflineIndicatorState.Online }
-	| { state: OfflineIndicatorState.Connecting }
-	| { state: OfflineIndicatorState.Synchronizing; progress: number }
-	| { state: OfflineIndicatorState.Offline; lastUpdate: Date | null; reconnectAction: () => void }
+	| { state: OfflineIndicatorState.Online; isMobile: boolean }
```

Comment:
I was thinking if we should just let it decide inside which one it is but maybe there will be cases where we want to decide it outside. It's also good when components are simple and don't decide on their own too much. Maybe we could rename it to `singleLine` or something like that?